docs/cri/registry.md

```diff
@@ -190,6 +190,51 @@ DEBU[0001] PullImageResponse: &PullImageResponse{ImageRef:sha256:78096d0a5478896
 Image is up to date for sha256:78096d0a54788961ca68393e5f8038704b97d8af374249dc5c8faec1b8045e42
 ```
 
+
+### Configure Registry Credentials Example - Dockerhub (docker.io)
+
+Confirm your dockerhub account works as expected
+pulling an image as follows:
+
+```console
+docker login -u <username> -p <password> 
+
+docker pull <user>/<image>:<tag>
+
+docker logout
+```
+
+
+Edit the containerd config (default location is at `/etc/containerd/config.toml`)
+to add your JSON key for dockerhub image pull requests:
+
+```toml
+version = 2
+
+[plugins."io.containerd.grpc.v1.cri".registry]
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
+    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
+      endpoint = ["https://registry-1.docker.io"]
```

Comment:
you don't have to explicitly configure this mirror endpoint for docker.io because we do that internally by default... and mirrors has been deprecated (replaced by the new hosts config so it will be removed at some point.

But I like it here as it explains, somewhat, that you need the auth for the actual endpoint.  

Maybe we should make the docker.io auth work for registry-1.docker.io based on suffix.

Reply:
will address the hosts comment later when we finalize what we are going to do with auth configs for these endpoints with respect to how we do host registry configs. Cheers!